tests/whisper_test.py

```diff
@@ -0,0 +1,103 @@
+import tempfile
+from typing import cast
+
+import jax
+import jax.numpy as jnp
+import numpy as onp
+from datasets import load_dataset
+from jax.random import PRNGKey
+from transformers import WhisperConfig as HfWhisperConfig
+from transformers import WhisperForConditionalGeneration as HfWhisperModel
+from transformers import WhisperProcessor
+
+import haliax as hax
+from haliax import Axis
+
+from levanter.compat.hf_checkpoints import RepoRef
+from levanter.models.whisper import WhisperConfig, WhisperModel
+from levanter.utils.tree_utils import inference_mode
+from test_utils import skip_if_no_torch
+
+
+def test_basic_forward_whisper():
```

Comment:
It doesn't! the WhisperProcessor and WhisperConfig both work without PyTorch (since I'm specifying the processor should return NP arrays rather than PT tensors)